common/src/main/java/com/sequenceiq/cloudbreak/common/gov/GovUtils.java

```diff
@@ -0,0 +1,30 @@
+package com.sequenceiq.cloudbreak.common.gov;
+
+import java.util.Comparator;
+import java.util.Set;
+
+import com.sequenceiq.cloudbreak.common.mappable.CloudPlatform;
+import com.sequenceiq.cloudbreak.common.type.Versioned;
+import com.sequenceiq.cloudbreak.util.VersionComparator;
+
+public class GovUtils {
```

Comment:
is there any reason this can't be a component? if it's a component then it could be easily mocked in tests